src/consensus/validation.h

```diff
@@ -161,12 +161,12 @@ static inline int64_t GetTransactionInputWeight(const CTxIn& txin)
 }
 
 /** Compute at which vout of the block's coinbase transaction the witness commitment occurs, or -1 if not found */
-inline int GetWitnessCommitmentIndex(const CBlock& block)
+inline int GetWitnessCommitmentIndex(const CTransaction& tx)
```

Comment:
605b372b1f62c4cd59d8056f0a6a1baf17c24014 nit: maybe rename `tx` to `coinbase`

Reply:
Renamed to `gentx` (the coinbase is the input only)